doc/man7/EVP_PKEY-EC.pod

```diff
@@ -2,7 +2,11 @@
 
 =head1 NAME
 
-EVP_PKEY-EC - EVP_PKEY EC keytype and algorithm support
+EVP_PKEY-EC,
+EVP_PKEY-ECDH,
+EVP_PKEY-ECDSA,
```

Comment:
Er...  as far as I know, there is no ECDH or ECDSA key type, so this sounds plain wrong to me.  It may be that certain curves are better suited for use with DH while others are better suited for DSA, but that's a different matter.  Whether a discussion on that should be in `EVP_PKEY-EC(7)` or `EVP_KEYEXCH-ECDH(7)` / `EVP_SIGNATURE-ECDSA(7)`, I cannot really say, but please do not claim in this way that there are separate ECDH and ECDSA keys, that's a rabbit hole OpenSSL got *out* of ages ago.

Reply:
Removed the tags.